test/normalize.html

```diff
@@ -0,0 +1,493 @@
+<!DOCTYPE html>
```

Comment:
Isn't this one of Normalize's tests? https://github.com/necolas/normalize.css/blob/master/test.html